doc/cephadm/operations.rst

```diff
@@ -483,30 +483,30 @@ flagged as an anomaly and a healthcheck (WARNING) state raised.
 
 CEPHADM_CHECK_SUBSCRIPTION
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
-This check relates to the status of vendor subscription. This check is
-performed only for hosts using RHEL, but helps to confirm that all hosts are
+This check relates to the status of OS vendor subscription. This check is
+performed only for hosts using RHEL and helps to confirm that all hosts are
 covered by an active subscription, which ensures that patches and updates are
 available.
 
 CEPHADM_CHECK_PUBLIC_MEMBERSHIP
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-All members of the cluster should have NICs configured on at least one of the
+All members of the cluster should have a network interface configured on at least one of the
 public network subnets. Hosts that are not on the public network will rely on
 routing, which may affect performance.
 
 CEPHADM_CHECK_MTU
 ~~~~~~~~~~~~~~~~~
-The MTU of the NICs on OSDs can be a key factor in consistent performance. This
+The MTU of the network interfaces on OSD hosts can be a key factor in consistent performance. This
 check examines hosts that are running OSD services to ensure that the MTU is
-configured consistently within the cluster. This is determined by establishing
+configured consistently within the cluster. This is determined by determining
 the MTU setting that the majority of hosts is using. Any anomalies result in a
-Ceph health check.
+health check.
 
 CEPHADM_CHECK_LINKSPEED
 ~~~~~~~~~~~~~~~~~~~~~~~
-This check is similar to the MTU check. Linkspeed consistency is a factor in
-consistent cluster performance, just as the MTU of the NICs on the OSDs is.
-This check determines the linkspeed shared by the majority of OSD hosts, and a
+This check is similar to the MTU check. Link speed consistency is a factor in
+consistent cluster performance, as is the MTU of the OSD node network interfaces.
+This check determines the links peed shared by the majority of OSD hosts, and a
```

Comment:
s/links peed/link speed/

But I enjoy the past tense of the verb "to pee" as much as the next guy.

(I'll take care of this in a force-push now. You don't have to do anything.)

Reply:
With certain NICs I've suffered ...

Reply:
It's wild how long it's taking for this PR to pass its tests. I've been waiting up for almost two hours to do the backports on this one. </vent>